### PR TITLE
Add automated deployment

### PR DESCRIPTION
**Pull request by KPLauritzen**
# Summary
- Add image tag to helm config
- Add teams notifier channel
- Add Helm chart to repo
- deploy with azure pipelines

### CODE REVIEW

**Comment by KPLauritzen** [comment]
See the pipeline working here: https://dfds.visualstudio.com/Smart%20Data/_build/results?buildId=549633&view=results

File: azure-pipelines.yml [hunk 1]
```diff
@@ -45,3 +49,63 @@ steps:
...
+      tokenPrefix: "$("
+      tokenSuffix: ")"
+
+  - task: HelmDeploy@0
+    displayName: helm upgrade grafana
+    inputs:
+      connectionType: 'Kubernetes Service Connection'
+      kubernetesServiceEndpoint: hellman-mlops-deploy
+      command: upgrade
+      # chartType: 'Name'
```
Comment (pacanada) [inline azure-pipelines.yml:85]: Dont we need that?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -45,3 +49,63 @@ steps:
...
+      # chartName: 'grafana/grafana'
+      chartType: 'FilePath'
+      chartPath: 'helm/grafana-6.51.2.tgz'
+      namespace: $(NAMESPACE)
+      valueFile: helm/values.yaml
+      arguments: '--set admin.existingSecret=grafana-password'
+      releaseName: grafana
+
+  - task: replacetokens@3
+    # Using replace tokens for the postgres data source conexions
```
Comment (pacanada) [inline azure-pipelines.yml:95]: This comment doesnt corresponds to the task

File: azure-pipelines.yml [hunk 3]
```diff
@@ -45,3 +49,63 @@ steps:
...
+      actionOnMissing: "fail"
+      keepToken: false
+      tokenPrefix: "$("
+      tokenSuffix: ")"
+
+  - task: HelmDeploy@0
+    displayName: helm upgrade grafana
+    inputs:
+      connectionType: 'Kubernetes Service Connection'
+      kubernetesServiceEndpoint: hellman-mlops-deploy
```
Comment (KPLauritzen) [inline azure-pipelines.yml:83]: marking two lines

File: azure-pipelines.yml (hunk 1 above)
Comment (KPLauritzen) [reply azure-pipelines.yml:85]: No. Remved

File: azure-pipelines.yml (hunk 2 above)
Comment (KPLauritzen) [reply azure-pipelines.yml:95]: Good eye. Removed

**Review by pacanada — APPROVED** [review]

**Pull request merged**